tests/Prep.m

```diff
@@ -0,0 +1,72 @@
+classdef Prep < matlab.unittest.TestCase
+    % Setup and teardown for tests.
+    properties (Constant)
+        CONN_INFO = struct(...
+            'host', dj.config('databaseHost'), ...
```

Comment:
Normally, tests are supposed to satisfy the self-contained criteria so all would-be contributors can run and verify. This breaks this notion by relying on a remote database server with proper access/permissions to run the test. We would normally run a local-only instance of a database server for the test suite. What if there are collisions while others are running the same test?